site/gdocs/components/Chart.tsx

```diff
@@ -105,7 +111,22 @@ export default function Chart({
                     border: "0px none",
                     height: d.height,
                 }}
-            />
+            >
+                {isExplorer ? (
```

Comment:
We don't have static images for explorers, and I'm not sure if it's worth it to make it work for them.